### PR TITLE
241 design screen

### PR DESCRIPTION
**Pull request by CedricCORGE**
New UI for design screen and update unit test for both screen

image
image
image

### CODE REVIEW

**Review by tbellicha — APPROVED** [review]
LGTM

File: Configurateur/front/lib/components/dialog/remove_design_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+                              RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(30.0),
+                              ),
+                            ),
+                          ),
+                          value: getClicked('Devant'),
+                          onChanged: (bool? value) {
+                            if (value == true) {
+                              setState(() {
+                                setClicked('Devant', true);
```
Comment (31Nathan) [inline Configurateur/front/lib/components/dialog/remove_design_dialog.dart:117]: setState(() {
    setClicked('Devant', value);
}

Comment (31Nathan) [reply Configurateur/front/lib/components/dialog/remove_design_dialog.dart:117]: Not important just to optimize

File: Configurateur/front/lib/components/dialog/remove_design_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+                              RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(30.0),
+                              ),
+                            ),
+                          ),
+                          value: getClicked('Gauche'),
+                          onChanged: (bool? value) {
+                            if (value == true) {
+                              setState(() {
+                                setClicked('Gauche', true);
```
Comment (31Nathan) [inline Configurateur/front/lib/components/dialog/remove_design_dialog.dart:185]: Here too, check the others if needed

File: Configurateur/front/lib/components/dialog/add_design_dialog.dart [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+                                      RoundedRectangleBorder>(
+                                    RoundedRectangleBorder(
+                                      borderRadius: BorderRadius.circular(30.0),
+                                    ),
+                                  ),
+                                ),
+                                value: getClicked('Derrière'),
+                                onChanged: (bool? value) {
+                                  if (value == true) {
+                                    setState(() {
```
Comment (31Nathan) [inline Configurateur/front/lib/components/dialog/add_design_dialog.dart:167]: Again check the files

**Pull request merged**